modules/cluster-logging-json-log-forwarding.adoc

```diff
@@ -0,0 +1,48 @@
+[id="cluster-logging-json-log-forwarding_{context}"]
+= JSON log forwarding
+
+Forwarding JSON logs gives fluentd, Elasticsearch, and other JSON-aware log collectors direct access to JSON log entries as JSON objects.
+
+To illustrate how this works, suppose that you have the following structured JSON log entry.
+
+.Example structured JSON log entry
+[source,yaml]
+----
+{"level":"info","name":"fred","home":"bedrock"}
+----
+
+Ordinarily, the `ClusterLogForwarder` custom resource (CR) forwards that log entry in the `message` field. The `message` field contains the JSON-quoted string equivalent of the JSON log entry, as shown in the following example.
+
+.Example `message` field
+[source,yaml]
+----
+{"message":"{\"level\":\"info\",\"name\":\"fred\",\"home\":\"bedrock\"",
+ "more fields..."}
+----
+
+To enable JSON log forwarding, you add `parse: json` to a pipeline in the `ClusterLogForwarder` CR, as shown in the following example.
```

Comment:
Is "you" strictly necessary here? Maybe I'm just used to "To do X, do Y" structure in other docs.

"as shown in the following example" might be considered a redundant phrase by some writers (here and 10 lines below). Will a colon be sufficient?

Reply:
This is more of a conceptual topic than a procedural one, so the phrase is descriptive rather than imperative/prescriptive.